ci/jenkins/utils.sh

```diff
@@ -14,6 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function check_and_cleanup_docker_build_cache() {
+    docker builder prune -af --filter="until=1h" > /dev/null
```

Comment:
you changed the filter from size-based to time-based?

Reply:
Replied in https://github.com/antrea-io/antrea/pull/5937#discussion_r1473665170